Sources/AppStoreConnectCLI/Arguments/BuildLocalizationInputArguments.swift

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+import FileSystem
+import Foundation
+
+struct BuildLocalizationInputArguments: ParsableArguments {
+    @Argument(help: "The locale information of the build localization resource. eg. (en-AU)")
+    var locale: String
+
+    @Option(
+        parsing: SingleValueParsingStrategy.scanningForValue,
+        help: """
+            A field that describes changes and additions to a build and indicates features you would like your users to test.
+            Please put the text inside an ''.
+            eg. ('Hello There')
+        """
+    )
+    var whatsNew: String?
+
+    @Option(help: "File path to a txt file that contains whats new text. (eg. whatsNew.txt)")
+    var path: String?
+}
+
+protocol CreateUpdateBuildLocalizationCommand {
+    var localization: BuildLocalizationInputArguments { get }
+}
+
+extension CreateUpdateBuildLocalizationCommand {
+
+    func validateWhatsNewInput() throws {
+        if localization.whatsNew != nil && localization.path != nil {
+            throw ValidationError("Please provide either a file path or whats new text in terminal.")
+        }
+    }
+
+    func readWhatsNew() throws -> String {
+        let whatsNew: String
+
+        if let whatsNewFromOption = localization.whatsNew {
+            whatsNew = whatsNewFromOption
+        } else if let filePath = localization.path {
+            guard let whatsNewString = try? String(contentsOfFile: filePath) else {
+                throw Error.invalidFilePath(filePath)
+            }
+
+            whatsNew = whatsNewString
```

Comment:
It's probably more idiomatic to use do catch here if we're wrapping the error:
```suggestion
            do {
                whatsNew = try String(contentsOfFile: filePath)
            } catch {
                throw Error.invalidFilePath(filePath)
            }
```

Alternatively if the errors that `String(contentsOfFile:)` throws seems descriptive enough you can lean on that error:
```suggestion
            whatsNew = try String(contentsOfFile: filePath)
```

Reply:
the second one looks good, no need for an extra enum Error, I've changed to that, please take a look 🙂